spec/greed_game_spec.rb

```diff
@@ -0,0 +1,56 @@
+require './greed_game'
+require './player'
+require './dice'
+
+RSpec.describe GreedGame do
+  let(:game) { GreedGame.new(2) }
+
+  describe '#initialize' do
+    it 'creates the correct number of players' do
+      expect(game.players.length).to eq(2)
+    end
+
+    it 'initializes players with the correct ids' do
+      expect(game.players.map(&:player_id)).to eq([1, 2])
+    end
+  end
+
+  describe '#calculate_score' do
+    it 'returns correct score for various dice rolls' do
+      expect(game.calculate_score([1, 1, 1, 5, 5])).to eq([1100, {}])
+      expect(game.calculate_score([2, 3, 4, 6, 2])).to eq([0, {2=>2, 3=>1, 4=>1, 6=>1}])
+      expect(game.calculate_score([1, 2, 3, 4, 5])).to eq([150, {2=>1, 3=>1, 4=>1}])
+    end
+  end
+
+  describe '#play_turn' do
+    before do
```

Comment:
Pls add more cases like multiple turns played

Reply:
not able to find a way to test this

Reply:
Let's discuss

Reply:
done